src/main/java/com/minecolonies/colony/workorders/WorkOrderBuild.java

```diff
@@ -152,6 +152,7 @@ public boolean isValid(Colony colony)
     public void attemptToFulfill(Colony colony)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "attemptToFulfill" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)